src/artm/messages.proto

```diff
@@ -27,16 +27,16 @@ message Item {
   optional int32 id = 1;
   repeated Field field = 2;  // obsolete in BigARTM v0.8.0
   optional string title = 3;
-  repeated int32 token_id = 4;  // obsolete
+  repeated int32 token_id = 4;
   repeated float token_weight = 5;
 
-  // new alternative to token_id field, containa ids of transaction tokens
-  repeated int32 transaction_token_id = 6;
+  // Field contains start index of each transaction from token_id.
+  // In case of regular text will be 0, 1, 2, 3 ... etc.
+  repeated int32 transaction_start_index = 6;
```

Comment:
transaction_start_index - it is convenient to use this variable in for loops like from transaction_start_index(i) to transaction_start_index(i+1) where i is index of transaction. This imply that the length of transaction_start_index should be "number of transactions plus one", and the last element should be "number of tokens". We use this pattern in CsrMatrix.